services/core-web/common/constants/actionTypes.js

```diff
@@ -78,6 +78,13 @@ export const STORE_COMPLIANCE_CODES = "STORE_COMPLIANCE_CODES";
 export const STORE_VARIANCE_STATUS_OPTIONS = "STORE_VARIANCE_STATUS_OPTIONS";
 export const STORE_VARIANCE_DOCUMENT_CATEGORY_OPTIONS = "STORE_VARIANCE_DOCUMENT_CATEGORY_OPTIONS";
 
+// Project Summaries
+export const STORE_PROJECT_SUMMARIES = "STORE_PROJECT_SUMMARIES";
+export const STORE_PROJECT_SUMMARY = "STORE_PROJECT_SUMMARY";
+export const STORE_PROJECT_SUMMARY_STATUS_OPTIONS = "STORE_PROJECT_SUMMARY_STATUS_OPTIONS";
+export const STORE_PROJECT_SUMMARY_DOCUMENT_CATEGORY_OPTIONS =
+  "STORE_PROJECT_SUMMARY_DOCUMENT_CATEGORY_OPTIONS";
```

Comment:
Same for 'STORE_PROJECT_SUMMARY_DOCUMENT_CATEGORY_OPTIONS' and 'STORE_PROJECT_SUMMARY_STATUS_OPTIONS' these should be handled already with the static content

Reply:
Ah okay, I think I misunderstood the usage pattern for these, but yeah I think you are right about me getting them into static content.